### PR TITLE
Move tracker options into `opts.tracker` (#649)

### PR DESCRIPTION
**Pull request by nkittsteiner**
Moving wtrc and rtcConfig opts to opts.tracker. Hope it helps :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,12 +68,14 @@ function WebTorrent (opts) {
...
+  self.tracker = opts.tracker !== undefined ? opts.tracker : {}
   self.torrents = []
   self.maxConns = Number(opts.maxConns) || 55
 
-  self._rtcConfig = opts.rtcConfig
-  self._wrtc = opts.wrtc || global.WRTC // to support `webtorrent-hybrid` package
+  if (typeof self.tracker === 'object') {
+    self.tracker.rtcConfig = opts.rtcConfig
+    self.tracker.wrtc = opts.wrtc || global.WRTC // to support `webtorrent-hybrid` package
+  }
```
Comment (feross) [inline index.js]: These options should be passed into the tracker constructor, not set directly on the tracker object like this.

Comment (feross) [inline index.js]: Nevermind.

**Comment by nkittsteiner** [comment]
Hi Feross, do you mean with nevermind leave it like this? Because I can fix that

**Comment by feross** [comment]
@nkittsteiner Thanks a lot for this PR. It illuminated a few issues that I hadn't though of, like how to handle `opts.rtcConfig` and `opts.wrtc`, as well as how to handle duplicate `getAnnounceOpts` functions passed into `opts.tracker` and also into the opts for `client.add`.

I'll send a PR that builds on yours.

**Comment by feross** [comment]
See: https://github.com/feross/webtorrent/pull/793

**Pull request closed without merging**